### DISCUSSION TITLE
江边沙地 | /posts/flying-water3/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#653] [Announcements]
https://yuanfan.rbind.io/posts/flying-water3/

**Comment by botsp** [comment]
新年快乐!

**Reply by earfanfan** [reply]
同乐同乐！我记得去年你也溜达过来给我说了新年快乐。希望大家都身体健康，日后每年都有机会互道新年快乐。

**Reply by botsp** [reply]
😀😀😀健康、如意~

**Reply by earfanfan** [reply]
我本来想与健康、如意对仗工整的话，该是平安、顺心。不过转念一想，其实我也可以画呀。请看，下面是我刚画好的。
![2024-12-30](https://github.com/user-attachments/assets/c42444f2-90af-4bcf-ba9c-3e3011d037e4)

**Reply by botsp** [reply]
画的很暖，哈哈，可以用相框抱起来吗。柿柿如意、好柿成真！

**Reply by earfanfan** [reply]
由于受到你这里提到相框抱起来的启发，后来我真的画了一幅小画用相框抱起来送人了。明年你也要记得给我说新年快乐哈，我都想好明年画的主题了，画绚烂的烟花。

**Comment by LFG12** [comment]
哇，白沙滩，和这个桥，好像武汉呀

**Reply by earfanfan** [reply]
就是武汉，俺就在这里。

**Reply by LFG12** [reply]
哈哈原来真是，俺也在武汉上学，有机会可以试着在武汉碰一下呀YF姐姐

**Reply by earfanfan** [reply]
可以呀，我堂妹依依这周末来看我，跟你一样是00后，我好奇你们小朋友碰面会说些什么，哈哈，不过我先问问她的意见。

**Reply by earfanfan** [reply]
依依说她也很好奇，那么后天周六要不要一起去解放公园看鸽子？

ps 我是这么问依依的：有胆量和勇气与陌生人尬聊不？依依说，我也很好奇，要是尴尬我们就跑路好不好。哈哈哈

**Reply by LFG12** [reply]
实在抱歉YF姐，我也很喜欢交朋友认识不同的人，到不怕尴尬，不过已经买了11号上午回家的车票了，实在不巧 等开学再到武汉了还有机会相遇，或者10号也有一天时间 第一次看到yf姐的博客，就能感受到是一个热爱生活，记录生活美好的人 在互联网的海洋中链接一个一个博客岛是一种奇妙的缘分 这是我的个人博客:lfg12.github.io，可以简单了解下我(不过博客还在装修，寒舍粗鄙厚着脸皮迎客了)

**Reply by LFG12** [reply]
下午4点看到消息后在qq邮箱回复了一下，结果发现可能是自动回复收不到，又在这里留下言

**Reply by LFG12** [reply]
哈哈“小朋友”这个词汇听起来好陌生了，有多久没有被听过叫我们小朋友的了哈哈，祝您天天开心，保持热爱

**Reply by earfanfan** [reply]
你的个人博客我昨天就去溜达了一圈，感觉装修很豪华呀。其实我当社畜太久了，忘记学生党还有寒假了，明天本社畜还要上班呐。那等你开学了再说吧（ps又在心中羡慕了一会）。

我比你大十几岁，所以确实看待你就像是看待小朋友一样。啊，我想起另一位小朋友了，戳[这里](https://guozheng.rbind.io/posts/2025/01/social-anthro-weekly-0/)，这位黄同学和你同岁，不过专业领域不同，他从本周六开始会写周刊，你有兴趣的话可以去溜达溜达。

对了，你的博客好像没有评论区，我想起很久以前，我的博客刚开张还没有弄好评论区的时候，我的第一位读者也吐槽过，可惜留不下脚印。

**Reply by earfanfan** [reply]
又及，我看到你的博客里有熊出没的页面，整理了每年大电影的时间线，但是只到2022年，其实我前些天刚看了逆转时空，觉得还蛮好看的，结尾还有些感动。

**Reply by LFG12** [reply]
哇，谢谢YF姐，我也是在诗涛哥的博客找到了您的博客，看到诗涛哥和二丫讲梵中都会写周报，正巧又发现您推荐的这个博客，我也想尝试一下及时写个周总结。
另，等我把熊出没的补完，延续到2025 ：）

**Reply by earfanfan** [reply]
其实可以不用称呼“您”，虽然咱们确实年龄相差大，但是并不需要被社会中的年龄秩序所影响。当然，如果这是你的习惯的话，也可以。

**Reply by LFG12** [reply]
哈哈好嘞YF姐，听你这么一说后才意识到原来自己已经不知不觉中已经成为了习惯，在YF姐的文字中总是感受到好似脱离开社会的温存，祝好

**Comment by think-crow** [comment]
昨天下午，偶遇同款三轮车：

<img src="https://cdn.jsdelivr.net/gh/think-crow/blog-creazy-server@master/public/uploads/beihaoqi-min.jpg" alt="倍好奇" width="60%" height="60%">

**Reply by earfanfan** [reply]
哈哈，我买的时候就知道这款很多老年人会买。

**Reply by think-crow** [reply]
讲真，你这喜好很特别，给人印象很深刻！

**Reply by earfanfan** [reply]
这不是喜好，是专门买来载阿木小狗出门玩的。

又及，我发现很多会骑自行车的人，很难学会骑三轮车。

**Reply by think-crow** [reply]
哦豁，那就是我喜好特别，你这选择我倒是越看越喜欢，哈哈哈！

>又及，我发现很多会骑自行车的人，很难学会骑三轮车。

还想着只有小孩会这样。记得小时候第一次骑（爷爷的大三轮）老往一边偏，把都把不住，后来学会了强行摆正，再后来就自然摆正。

**Reply by earfanfan** [reply]
我猜是因为自行车是要靠人来掌控平衡和方向，而三轮车本身是平衡的只需要人掌控方向，但是会骑自行车的一旦骑三轮车会不自觉的以为歪了，进一步会不自觉地把方向也调歪，但三轮车转向没有自行车灵活，于是就变成自己使劲往一边偏了。
